projects/istio/build.sh

```diff
@@ -15,6 +15,15 @@
 #
 ################################################################################
 
+cd $SRC
+git clone --branch=dev https://github.com/AdamKorcz/go-118-fuzz-build $SRC/go-118-fuzz-build
+cd $SRC/istio
+go mod edit -replace github.com/AdamKorcz/go-118-fuzz-build=/src/go-118-fuzz-build
```

Comment:
Can you add TODOs here in these project build.sh scripts to remove once upstreamed?

Reply:
Created this instead: https://github.com/istio/istio/pull/42028